src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -159,15 +166,55 @@ constexpr Containers::StringView sceneFieldNames[] = {
     "Visibility"_s,
 };
 
-struct MeshChunk {
-    uint32_t meshId;
-    uint32_t meshMaterialIndex;
-};
+Containers::StringView blendModeToString(ufbx_blend_mode mode) {
+    switch (mode) {
+    /* LCOV_EXCL_START */
+    case UFBX_BLEND_TRANSLUCENT: return "translucent"_s;
+	case UFBX_BLEND_ADDITIVE: return "additive"_s;
+	case UFBX_BLEND_MULTIPLY: return "multiply"_s;
+	case UFBX_BLEND_MULTIPLY_2X: return "multiply2x"_s;
+	case UFBX_BLEND_OVER: return "over"_s;
+	case UFBX_BLEND_REPLACE: return "replace"_s;
+	case UFBX_BLEND_DISSOLVE: return "dissolve"_s;
+	case UFBX_BLEND_DARKEN: return "darken"_s;
+	case UFBX_BLEND_COLOR_BURN: return "colorBurn"_s;
+	case UFBX_BLEND_LINEAR_BURN: return "linearBurn"_s;
+	case UFBX_BLEND_DARKER_COLOR: return "darkerColor"_s;
+	case UFBX_BLEND_LIGHTEN: return "lighten"_s;
+	case UFBX_BLEND_SCREEN: return "screen"_s;
+	case UFBX_BLEND_COLOR_DODGE: return "colorDodge"_s;
+	case UFBX_BLEND_LINEAR_DODGE: return "linearDodge"_s;
+	case UFBX_BLEND_LIGHTER_COLOR: return "lighterColor"_s;
+	case UFBX_BLEND_SOFT_LIGHT: return "softLight"_s;
+	case UFBX_BLEND_HARD_LIGHT: return "hardLight"_s;
+	case UFBX_BLEND_VIVID_LIGHT: return "vividLight"_s;
+	case UFBX_BLEND_LINEAR_LIGHT: return "linearLight"_s;
+	case UFBX_BLEND_PIN_LIGHT: return "pinLight"_s;
+	case UFBX_BLEND_HARD_MIX: return "hardMix"_s;
+	case UFBX_BLEND_DIFFERENCE: return "difference"_s;
+	case UFBX_BLEND_EXCLUSION: return "exclusion"_s;
+	case UFBX_BLEND_SUBTRACT: return "subtract"_s;
+	case UFBX_BLEND_DIVIDE: return "divide"_s;
+	case UFBX_BLEND_HUE: return "hue"_s;
+	case UFBX_BLEND_SATURATION: return "saturation"_s;
+	case UFBX_BLEND_COLOR: return "color"_s;
+	case UFBX_BLEND_LUMINOSITY: return "luminosity"_s;
+	case UFBX_BLEND_OVERLAY: return "overlay"_s;
+    default:
+        Warning{} << "Unhandled blend mode" << Int(mode);
+        return {};
+    /* LCOV_EXCL_STOP */
```

Comment:
> Otherwise, if there's some `UFBX_BLEND_INVALID`

If only... There's two sentinel values that I'd love to get rid of `UFBX_BLEND_MODE_COUNT` which could just be outside of the enum and validated with a little linter, and the worse one `UFBX_BLEND_MODE_FORCE_32BIT` which is required for having a stable ABI. It's super annoying that you can't do exhaustive enum checks because of that, I'll need to look if there's some method of forcing enum size on the big three and maybe I could have those filler values only for weird compilers like TCC.

Reply:
Sentinels should be fine, just add another unreachable case with them:

```cpp
case UFBX_BLEND_MODE_COUNT:
case UFBX_BLEND_MODE_FORCE_32BIT:
    CORRADE_INTERNAL_ASSERT_UNREACHABLE(); /* LCOV_EXCL_LINE */
```

Huh, C didn't get the C++11 typed enum feature yet? You could do this maybe:

```cpp
typedef enum ufbx_blend_mode
    #ifdef __cplusplus
    : std::uint32_t
    #endif
{
    ...
    #ifndef __cplusplus
    UFBX_BLEND_MODE_FORCE_32BIT = 0x7fffffff
    #endif
```

Bad suggestion, sorry :D